cms/jinja2/templates/pages/statistical_article_page.html

```diff
@@ -28,12 +28,27 @@
     {% endif %}
 
     {% if page.is_latest %}
-        {% set release_html -%}
-            <a href="{{ routablepageurl(page.get_parent().specific, "previous_releases") }}">{{ _("View previous releases") }}</a>
-        {%- endset %}
+        {% if latest_version_url %}
+            {% set infoPanelText = _("This version has been amended since publication") %}
+            {% set releases_text = _("View amended version") %}
+            {% set infoPanelType = 'ons-red' %}
+            {% set releases_url -%}
+                {{ latest_version_url }}
+            {%- endset %}
+        {% else %}
+            {% set infoPanelText = _("Latest release") %}
+            {% set releases_text = _("View previous releases") %}
+            {% set infoPanelType = 'ons-green' %}
+            {% set releases_url -%}
+                {{ routablepageurl(page.get_parent().specific, "previous_releases") }}
+            {%- endset %}
```

Comment:
I think this should work:

```suggestion
            {%- set releases_url = routablepageurl(page.get_parent().specific, "previous_releases") -%}
```